packages/noya-renderer/src/components/EditablePath.tsx

```diff
@@ -51,6 +51,47 @@ function EditablePathPoint({ point, fill, stroke }: EditablePathPointProps) {
   );
 }
 
+interface EditableControlPointProps {
+  point: Point;
+  controlPoint: Point;
+  fill: Paint;
+  stroke: Paint;
+}
+
+function ControlPathPoint({
```

Comment:
This component is rendering 2 things, the control point and the connecting line, but I think it'd be clearer with just 1. Can we try moving the `Polyline` into the parent component, `EditablePath`?

E.g. within the 2 `.maps`:

```tsx
<Fragment key={index}>
  <Polyline
    points={[point.point, point.curveFrom]}
    paint={stroke}
  />
  <ControlPathPoint ... />
</Fragment>
```

Reply:
Also probably rename `ControlPathPoint` => `EditablePathControlPoint` and `EditableControlPointProps` => `EditablePathControlPointProps`, for consistency in this file

Reply:
Done.